### PR TITLE
add verbs to built fetcher

### PR DESCRIPTION
**Pull request by markusn**

### CODE REVIEW

**Comment by andolf** [comment]
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
 Changelog
 =========
 
+# 5.2.0
+
+Added the following methods to the object returned from `fetchBuilder`:
+
+* `get` - makes a GET request using the behavior passed to fetchBuilder´
```
Comment (andolf) [inline CHANGELOG.md:8]: markdown syntax should be wonky with the trailing `´`

File: README.md [hunk 2]
```diff
@@ -51,6 +51,20 @@ fetch(options, function (err, content) {
 });
 ```
 
+#### Using one build with many verbs
+
+```javascript
+const behavior = {};
+const request = require("exp-fetch")(behavior);
+
```
Comment (guli-bn) [inline README.md:59]: Worth mentioning that `behavior.httpMethod` will be overruled when using these to prevent any confusion?

**Review by guli-bn — APPROVED** [review]
Looks great! 😍

**Review by mgronblad — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (markusn) [reply README.md:59]: fixing

File: CHANGELOG.md (hunk 1 above)
Comment (markusn) [reply CHANGELOG.md]: fixing

**Pull request merged**